src/Domain/SendEmailCommand.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace MatchBot\Domain;
+
+/**
+ * @psalm-type emailParams array<string,string|null|int|float|array>
+ */
+readonly class SendEmailCommand
```

Comment:
I'm wondering if there's a slight semantic clash here, with Symfony and existing MatchBot conventions of using 'Command' mostly to mean console commands?

Personally I would also maybe put it somewhere more specific to how it's called than `Domain` – perhaps the namespace could be `Application\Notify` or something and the class could be `Email`?

Reply:
Yep I was thinking it might be better to call it email or something like that, and then change the function name so we do `$mailer->send($email)` instead of `$mailer->accept($command)`. I think I was thinking of it as domain since it's only depending on I/O free domain stuff like donation, but there's no rule that we can't have classes like that anywhere else. Will see where it could go.

Reply:
Probably influenced because we just read this refactoring in the book in my book club https://refactoring.com/catalog/replaceFunctionWithCommand.html